hack/source-manifests.sh

```diff
@@ -25,9 +25,12 @@ if [ -z "$NOOBAA_IMAGE" ] || [ -z "$ROOK_IMAGE" ]; then
 fi
 
 # always start fresh and remove any previous artifacts that may exist.
+rm -rf $OCS_FINAL_DIR
```

Comment:
adding `rm -rf $OCS_FINAL_DIR` here, we can also remove this from `hack/generate-unified-csv.sh`